corehq/ex-submodules/casexml/apps/phone/dbaccessors/sync_logs_by_user.py

```diff
@@ -22,7 +30,20 @@ def get_last_synclog_for_user(user_id):
         return properly_wrap_sync_log(row['doc'])
 
 
-def get_synclogs_for_user(user_id, limit=10):
+def get_synclogs_for_user(user_id, limit=10, wrap=True):
+    synclogs = SyncLogSQL.objects.filter(user_id=user_id).order_by('date')[:limit]
+    if synclogs:
```

Comment:
If a user has only 1 synclog in SQL and 9 in couch then this will only return the 1 SQL synclog instead of the 10 defined in limit. I think that's probably ok for now